### PR TITLE
Storing country selection

### LINKED ISSUE #12: Store country selection in local web storage

### PR DESCRIPTION
**Pull request by bancho22**
Closes #12

### CODE REVIEW

**Comment by bancho22** [comment]
After merging #18, conflicts should be resolved with _main_

**Comment by bence-toth** [comment]
> After merging #18, conflicts should be resolved with _main_

You could have just branched off that one :see_no_evil:

**Comment by bancho22** [comment]
> > After merging #18, conflicts should be resolved with _main_
> 
> You could have just branched off that one 🙈

Where's the fun in that? 😆

**Comment by bence-toth** [comment]
#18 is now merged to `main`

**Comment by bancho22** [comment]
Great. I'll look into the conflicts now.

File: src/App.js [hunk 1]
```diff
@@ -3,20 +3,42 @@ import {useState} from 'react'
 import countries from './countries'
 import getCountryData from './consumer'
 import geolocationStates from './geolocationStates'
-import {useGoogleCharts, useGeolocation, useChartUpdate, useResizeListener} from './hooks'
+import {useCountrySelectionStore, useGeolocation, useChartUpdate, useResizeListener, useGoogleChartSetUp} from './hooks'
```
Comment (bence-toth) [inline src/App.js:6]: This time would probably be as good as any to run the js files through prettier :slightly_smiling_face:

File: src/hooks.js [hunk 2]
```diff
@@ -65,29 +68,84 @@ const useResizeListener = ({data, selectedCountries, geolocationState}) => {
...
 
-const useGoogleCharts = () => {
+const useCountrySelectionStore = () => {
+  const [storedCountries, setStoredCountries] = useState([])
+
   useEffect(() => {
-    google.charts.load('current', {
-      packages: ['corechart']
-    })
+    const countries = localStorage.getItem('storedCountries')
```
Comment (bence-toth) [inline src/hooks.js:114]: Nit: Maybe call it `selectedCountries` instead. You could prefix literally everything with `stored` in local storage :joy_cat:

**Review by bence-toth — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (bancho22) [reply src/App.js]: I fixed this line. Haven't run the file through a prettier tho. I don't think I have it set up. Feel free to do it if needed.

File: src/hooks.js (hunk 2 above)
Comment (bancho22) [reply src/hooks.js]: Sure. Makes sense.

**Comment by bancho22** [comment]
Conflicts have been resolved.

File: src/App.js (hunk 1 above)
Comment (bence-toth) [reply src/App.js]: I applied prettier

**Review by bence-toth — APPROVED** [review]
Made a few minor changes (storing initial selection in local storage, falling back to country if iso doesn't match a known slug).

LGTM now! :100:

**Pull request merged**